config/routes.rb

```diff
@@ -6,7 +6,7 @@
 
   resources :users, only: [:index, :show] do
     resources :posts, only:[:index, :show, :new, :create, :destroy] do
-      resources :comments, only:[:new, :create, :destroy]
+      resources :comments, only:[:index, :new, :create, :destroy]
       resources :likes, only:[:create]
     end
   end
```

Comment:
- Kindly make sure you have
   -  API endpoint that returns all comments for a post - **GET :** `/users/:user_id/posts/:id/comments`
   -  API endpoint that adds' a comment to a post - **POST :** `/users/:user_id/posts/:id/comments`
and your endpoints are working as expected. Please the your them before submission and give the documentation in the pull request on how your endpoints are working :+1: